filebeat/Makefile

```diff
@@ -34,9 +34,19 @@ modules:
 	rm -rf _meta/module.generated
 	rsync -av module/ _meta/module.generated --exclude "_meta" --exclude "*/*/test"
 
+# Collects all module configs
+.PHONY: configs
+configs: python-env
+	cat ${ES_BEATS}/filebeat/_meta/common.p1.yml > _meta/beat.yml
+	. ${PYTHON_ENV}/bin/activate; python ${ES_BEATS}/metricbeat/scripts/config_collector.py --beat ${BEAT_NAME} $(PWD) >> _meta/beat.yml
+	cat ${ES_BEATS}/filebeat/_meta/common.p2.yml >> _meta/beat.yml
+	cat ${ES_BEATS}/filebeat/_meta/common.full.p1.yml > _meta/beat.full.yml
+	. ${PYTHON_ENV}/bin/activate; python ${ES_BEATS}/metricbeat/scripts/config_collector.py --beat ${BEAT_NAME} --full $(PWD) >> _meta/beat.full.yml
+	cat ${ES_BEATS}/filebeat/_meta/common.full.p2.yml >> _meta/beat.full.yml
```

Comment:
👍  on the naming. Was just curious as I think I will potentially use the same in metricbeat ...